release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3319,6 +3319,56 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+[id="ocp-4-14-26-bug-fixes"]
+==== Bug fixes
+
+* Previously, when the `ContainerRuntimeConfig` resource was created as an extra manifest for single-node {product-title} installation, the bootstrap failed with the following error message: "more than one ContainerRuntimeConfig found that matches MCP labels". With this release, the incorrect processing of the `ContainerRuntimeConfig` resource is fixed and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-30153[*OCPBUGS-30153*])
+
+* Previously, an issue with NodePort traffic-forwarding caused the Transmission Control Protocol (TCP) traffic to be directed to pods under a terminating state. With this release, the endpoints selection logic fully implements `KEP-1669 ProxyTerminatingEndpoints` and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-32319[*OCPBUGS-32319*])
+
+* Previously, for {product-title} deployments on {rh-openstack-first}, the `MachineSet` object did not correctly apply the value for the `Port Security` parameter. With this release, the `MachineSet` object applies the `port_security_enabled` flag as expected. (link:https://issues.redhat.com/browse/OCPBUGS-32428[*OCPBUGS-32428*])
+
+* Previously, static Persistent Volumes in Azure File on Workload Identity clusters could not be configured due to an issue with the driver. With this release, the issue has been resolved and static Persistent Volumes mount correctly. (link:https://issues.redhat.com/browse/OCPBUGS-33039[*OCPBUGS-33039*])
+
+* Previously, the load-balancing algorithm had flaws that led to increased memory usage and a higher risk of excessive memory consumption. With this release, the service filtering logic for load-balancing is updated and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33389[*OCPBUGS-33389*])
+
+* Previously, the Ironic Python Agent (IPA) failed when trying to wipe disks because it expected the wrong byte sector size, which caused the node provisioning to fail. With this release, the IPA checks the disk sector size and node provisioning succeeds. (link:https://issues.redhat.com/browse/OCPBUGS-33452[*OCPBUGS-33452*])
+
+* Previously, attempting to remove an alternate service when editing a Route by using the form view did not remove the alternate service from the Route. With this update, the alternate service is removed and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33462[*OCPBUGS-33462*])
+
+* Previously, the `vsphere-problem-detector` operator could not connect to vCenter because the operator did not have the HTTP(S) proxy configured. With this release, the `vsphere-problem-detector` operator uses the same HTTP(S) proxy as the rest of the cluster and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33467[*OCPBUGS-33467*])
```

Comment:
```suggestion
* Previously, the `vsphere-problem-detector` Operator could not connect to vCenter because the operator did not have the HTTP(S) proxy configured. With this release, the `vsphere-problem-detector` Operator uses the same HTTP(S) proxy as the rest of the cluster and the issue has been resolved. (link:https://issues.redhat.com/browse/OCPBUGS-33467[*OCPBUGS-33467*])
```
See the [repo glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#operator)